.github/workflows/generate_and_push_crowdin_translations.yml

```
@@ -27,7 +27,8 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - name: Checkout
         uses: actions/checkout@v4
-
+        with: 
+          ssh-key: ${{ secrets.ACTIONS_PAT }}
```

```suggestion
          token: ${{ secrets.ACTIONS_PAT }}
```

I think this is `token` instead of `ssh-key`